### ISSUE TITLE
phone number formating

### ISSUE THREAD

**Issue by braven112** [AlaskaAirlines/auro-input#78]
## Is your feature request related to a problem? Please describe.

The account team will is looking to add phone number support to the auro-input

## Describe the solution you'd like

![image](https://user-images.githubusercontent.com/5118118/142090687-1ce72f33-638d-42e2-94c3-a0a233c3e082.png)

## Describe alternatives you've considered

No formatting

## Additional context

Reach out to Shelby for questions.

**Comment by blackfalcon** [comment]
If there is a need for this, I would like to piggyback this work onto the updates needed for date format support as well. That is an update that is needed prior to the re-development of the datepicker element. 

Both of these are things supported by https://nosir.github.io/cleave.js/

**Comment by blackfalcon** [comment]
Updating this to be a design story for follow up with @leeejune. Once the design is complete, a new issue will be created for the feature work. 

Removing from release candidate milestone until this is decided to be a feature to work on.
